packages/components/src/micro_helmet/index.stories.jsx

```diff
@@ -3,7 +3,7 @@ import MicroHelmet from './index';
 import MicroHelmetProvider from './provider';
 
 const usePageTitle = () => {
-  const [title, setTitle] = useState(document.title);
+  const [title, setTitle] = useState(typeof document !== 'undefined' && document.title);
```

Comment:
default title value is `false`, is that what you wanted?

Reply:
Yes